### ISSUE TITLE
Question about the transformer_encoder

### ISSUE THREAD

**Issue by TitleZ99** [LCFractal/TGDT#1]
Thanks for sharing this great work. I have some questions about it. First, you mention you use pre-extracted image features but you choose use transformer to extract image features instead. This dosen't match the paper. Second, i am wondering why you add transformer_encoder_text and transformer_encoder_img. I saw your baseline TERAN before. Compared with it, you add these two encoders. I can't get it, especiallly the operation of .detach(). Can you help me solve these two questions? Thanks again for this work!
![1693890408075](https://github.com/LCFractal/TGDT/assets/96060205/34897974-92ac-4628-8ac5-4d40cb46e2e1)
